clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -272,6 +267,8 @@ public HubConnectionState getConnectionState() {
 
     /**
      * Stops a connection to the server.
+     * @param errorMessage An error message if the connected needs to be stopped because of an error.
```

Comment:
This is private actually... doesn't technically need any Javadocs ;). It's fine to leave it, but I was about to nit pick the comment and realized it was private.

Reply:
Lol. Good point